web/themes/custom/cloudy/scss/components/_form.scss

```diff
@@ -95,4 +95,36 @@ span.prev-link {
 }
 span.next-link .fas {
   margin: 0 0 0 .5rem;
+}
+
+// admin content views, make more compact, powr-893
+
+.view-admin-group-content .form-check {
+  padding-left: 0;
+}
+.view-admin-group-content .dropbutton-multiple {
+  margin-right: 2em !important;
+  padding-right: 3em !important;
+}
+.view-admin-group-content table {
+  font-size: 0.8em;
```

Comment:
We should probably use the $font-size-sm variable rather than hardcoding this. We have responsive fonts and this wouldn't get hit by those rules. (I'm less concerned about the padding and margin.)

Reply:
Did you try the variable instead of hard coding an em size? How did it look?